src/App.jsx

```diff
@@ -1,41 +1,65 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 
-const App = () => (
-  <div className="App">
-    <header className="App__header">
-      <label>
-        Select a user: &nbsp;
+const App = () => {
+  const [posts, setPosts] = useState([]);
+  const [selectedUser, setSelectedUser] = useState('0');
```

Comment:
```suggestion
  const [selectedUser, setSelectedUser] = useState(null);
```
use null as default value

Reply:
+